gossipsub.go

```diff
@@ -342,15 +342,25 @@ func (gs *GossipSubRouter) sendRPC(p peer.ID, out *RPC) {
 		return
 	}
 
-	select {
-	case mch <- out:
-	default:
-		log.Infof("dropping message to peer %s: queue full", p)
-		// push control messages that need to be retried
-		ctl := out.GetControl()
-		if ctl != nil {
-			gs.pushControl(p, ctl)
+	if out.Size() > maxRPCSize {
```

Comment:
here too, this check seems unnecessary.

Reply:
I'm worried we could add too much gossip. That's why I have that check. But I can remove it if you think it's overkill.

Reply:
but the check is done before we even add the gossip!

Reply:
ok, this check is reasonable on second thought, if we are adding too much gossip (over 64KB) we probably want to know.